parser_test.go

```diff
@@ -110,23 +111,35 @@ func TestParse(t *testing.T) {
 			continue
 		}
 
-		tags, err := Parse(testCase.filename, testCase.relative, basepath)
-		if err != nil {
-			t.Errorf("[%s] Parse error: %s", testCase.filename, err)
-			continue
-		}
+		// Repeat each test - once with all symbols and once without private symbols
+		for _, excludePrivate := range []bool{false, true} {
```

Comment:
You could also just add an extra file in `tests/`, add a `excludePrivate` as a field in the `testCases` struct for this specific option and then test if the correct (exported only) tags are returned.

I understand why you'd want to do it this way, as this will check all the different types of tags in all the testfiles. However, I think I'd prefer not to treat this option as a special case compared to the other tests. What do you think?